server/embed/config.go

```diff
@@ -464,7 +470,7 @@ type securityConfig struct {
 	ClientKeyFile  string `json:"client-key-file"`
 	CertAuth       bool   `json:"client-cert-auth"`
 	TrustedCAFile  string `json:"trusted-ca-file"`
-	AutoTLS        bool   `json:"auto-tls"`
+	AutoTLS        bool   `json:""`
```

Comment:
Is this expected?

Reply:
Oh I'm not sure how this happened. Thanks for pointing it out.